src/ImageClassifier/darknet.js

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+import * as tf from '@tensorflow/tfjs';
+import IMAGENET_CLASSES_DARKNET from '../utils/IMAGENET_CLASSES_DARKNET';
+
+const DEFAULTS = {
+  DARKNET_URL: 'https://rawgit.com/TheHidden1/ml5-data-and-models/darknetclassifier/models/darknetclassifier/darknetreference/model.json',
```

Comment:
Just a reminder that we need to change this to the ones hosted in the ml5.js models

Reply:
i made a pull request on https://github.com/ml5js/ml5-data-and-models/pull/32
can you merge it ?